libdrgn/type_index.c

```diff
@@ -161,49 +161,49 @@ static void default_primitive_types_init(void)
 
 	drgn_int_type_init(&default_primitive_types[DRGN_C_TYPE_CHAR],
 			   drgn_primitive_type_spellings[DRGN_C_TYPE_CHAR][0],
-			   1, true);
+			   1, true, NULL);
```

Comment:
These primitive types are all for C, so let's explicitly pass `&drgn_language_c`.